### ISSUE TITLE
Incorrect "top" link generated in HTML

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1584]
I have in my conf.py file:

```
#!python

master_doc = 'manual'
```

When I run "make html" the top file of the generated documentation is "manual.html", which is correct, but all HTML files contain this in the head:

```
#!html

    <link rel="top" title="Project Title" href="index.html" />
```

and index.html doesn't even exist. The link generated in the body of the page is however correct.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1584
- Originally reported by: Ignacio Fernández Galván
- Originally created at: 2014-10-01T19:49:43.765

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-01 17:59:38+00:00_

Closes #1584: Point to master doc in HTML "top" link.

→ <<cset ee245c7bf15b601c5369475ea1ab0854f39765f9>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-01 17:59:49+00:00_

Thanks for the report!
